package.json

```diff
@@ -9,12 +9,16 @@
   "scripts": {
     "postinstall": "yarn build",
     "build": "lerna run build",
-    "start-api": "cd packages/api; yarn start"
+    "start-api": "cd packages/api; yarn start",
+    "start-api-win": "cd packages/api && yarn start"
```

Comment:
Change this over to a lerna run. EG: `lerna run start --scope @code-mentoring/api` or something like that. Check the docs for lerna. That way we only have one command here.